features/reordering_previously_placed_order.feature

```diff
@@ -0,0 +1,55 @@
+@managing_orders @todo
+Feature: Reordering previously placed order
+    In order to reorder the same order as I placed before
+    As a Customer
+    I want to have items' quantity and prices recalculated during reordering process
+
+    Background:
+      Given the store operates on a single channel in the "United States" named "Web"
+      And the store has a product "Angel T-Shirt" priced at "$39.00"
+      And the store ships everywhere for free
+      And the store allows paying with "Cash on Delivery"
+      And I am a logged in customer
+      And there is a promotion "Order's Extravaganza"
+      And this promotion gives "$20.00" discount to every order
+      And I placed an order "#00000666"
+      And I bought a single "Angel T-Shirt"
+      And I addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States"
+      And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
+      And I chose "Free" shipping method with "Cash on Delivery" payment
+
+    Scenario: Reordering previously placed order
+      Given I browse my orders
+      When I click reorder button next to the order "#00000666"
+      Then I should be on the checkout summary step
```

Comment:
`When ... And ... Then ...`

Reply:
And didn't we agree that we should be redirected to cart page rather than checkout summary page?